src/liboslexec/pointcloud.cpp

```diff
@@ -470,22 +471,28 @@ osl_pointcloud_write_helper(ustringhash_pod* names_, TypeDesc* types,
 
 
 
-OSL_SHADEOP int
-osl_pointcloud_write(ShaderGlobals* sg, ustringhash_pod filename_,
-                     const Vec3* pos, int nattribs,
-                     const ustringhash_pod* names_, const TypeDesc* types,
-                     const void** values)
+OSL_SHADEOP OSL_HOSTDEVICE int
+osl_pointcloud_write(OpaqueExecContextPtr oec, ustringhash_pod filename_,
+                     const void* pos_, int nattribs, const void* names_,
+                     const void* types_, const void* values_)
 {
+#ifndef __CUDA_ARCH__
+    ShaderGlobals* sg = (ShaderGlobals*)oec;
     ShadingSystemImpl& shadingsys(sg->context->shadingsys());
     if (shadingsys.no_pointcloud())  // Debug mode to skip pointcloud expense
         return 0;
```

Comment:
Does anyone need/use the no_pointcloud option?
We could support it via the ShadingStateUniform, but don't want to if there are no use cases for it.